app/plugin.go

```diff
@@ -459,8 +465,8 @@ func (a *App) GetMarketplacePlugins(filter *model.MarketplacePluginFilter) ([]*m
 	return result, nil
 }
 
-func (a *App) getRemotePlugins(filter *model.MarketplacePluginFilter) ([]*model.MarketplacePlugin, *model.AppError) {
-	var result []*model.MarketplacePlugin
+func (a *App) getRemotePlugins(filter *model.MarketplacePluginFilter) (map[string]*model.MarketplacePlugin, *model.AppError) {
```

Comment:
github won't let me comment on the line where we're returning apperrors, but we should update func name in the errors to be `getRemotePlugins`

Reply:
@ali-farooq0 Could you point me where that is? I'm not sure what you are referring to

Reply:
@marianunez 
https://github.com/mattermost/mattermost-server/pull/13433/files#diff-d07092579ced8908e9521d5930e3328aR473
https://github.com/mattermost/mattermost-server/pull/13433/files#diff-d07092579ced8908e9521d5930e3328aR481
https://github.com/mattermost/mattermost-server/pull/13433/files#diff-d07092579ced8908e9521d5930e3328aR490

basically all the places where we return `NewAppError` in this function

Reply:
Gotcha, done!